docs/content/operator/gas-station/architecture/components.mdx

```diff
@@ -0,0 +1,151 @@
+---
+title: Gas Station Components
+description: Learn about the key components of the IOTA Gas Station
+teams:
+  - iotaledger/identity
+tags:
+  - gas-station
+  - sponsored-transactions
+  - explanation
+  - reference
+---
+
+The **IOTA Gas Station** consists of several key components, each playing a crucial role in managing transactions efficiently and securely.
+
+```mermaid
+flowchart LR
+ subgraph subGraph0["Gas Station Pool"]
+        GSS["Gas Station Servers"]
+  end
+    KMS[("Key Management Store")] -.-> GSS
+    GSS --> REDIS[("Persistent Storage (Redis)")]
+
+
+    style KMS stroke-width:1px,stroke-dasharray: 5
+```
+
+##  Gas Station Server
+
+The **Gas Station Server** is the central component of the system. Each deployment of a Gas Station Server is associated with a single sponsor wallet, responsible for funding transactions. To scale operations, multiple Gas Station Servers can be deployed together as a **Gas Station Pool**, allowing them to share the same sponsor address and persistent storage.
+
+### HTTP Endpoints:
```

Comment:
Consider linking the endpoints to the API description

Reply:
good point, fixed